src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java

```diff
@@ -0,0 +1,49 @@
+package me.nikl.cookieclicker.updates.factory;
+
+import me.nikl.cookieclicker.Game;
+import me.nikl.gamebox.util.NumberUtil;
+import me.nikl.cookieclicker.buildings.Buildings;
+import me.nikl.cookieclicker.updates.Upgrade;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.material.MaterialData;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Niklas on 09.07.2017.
+ *
+ */
+public class ChineseWorkers extends Upgrade{
+
+    public SturdierConveyorBelts(Game game) {
+        super(game, 13);
```

Comment:
And the ID has to be unique. Simplest would be to use 1000. I am using the official IDs from the original game and they go up to a few hundred.